README.rst

```diff
@@ -75,197 +77,186 @@ Installation
+Now enable shell completion:
 
 .. code-block:: bash
 
     echo "autoload -U compinit; compinit" >> ~/.zshrc
 
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 OS X
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~
 
 If you install ``cartridge-cli`` from ``brew``, it automatically installs both
-Bash and Zsh completions.
+Bash and Zsh completion.
 
--------------------------------------------------------------------------------
 Usage
--------------------------------------------------------------------------------
+-----
 
-For more details, say:
+For more details, use the ``--help`` flag:
 
 .. code-block:: bash
 
    cartridge --help
 
-The following commands are supported:
-
-* ``create`` — create a new application from template;
-* ``build`` — build the application for local development and testing;
-* ``start`` — start a Tarantool instance(s);
-* ``stop`` — stop a Tarantool instance(s);
-* ``status`` — get current instance(s) status;
-* ``log`` — get logs of instance(s);
-* ``clean`` - clean instance(s) files;
-* ``pack`` — pack the application into a distributable bundle;
-* ``repair`` — patch cluster configuration files;
-* `admin <doc/admin.rst>`_ - call an admin function provided by the application;
-* `replicasets <doc/replicasets.rst>`_ - manage cluster replica sets running locally;
-* `enter and connect <doc/connect.rst>`_ - connect to running instance;
-* `failover <doc/failover.rst>`_ - manage cluster failover.
-
-The following global flags are supported:
-
-* ``verbose`` — verbose mode, additional log messages are shown as well as
-  commands/docker output (such as `tarantoolctl rocks make` or `docker build` output);
-* ``debug`` — debug mode (the same as verbose, but temporary files and
-  directories aren't removed);
-* ``quiet`` — the mode that hides all logs; only errors are shown.
-
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-An application lifecycle
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Here is a list of supported Cartridge CLI commands:
+
+* ``create``: create a new application from template.
+* ``build``: build an application for local development and testing.
+* ``start``: start one or more Tarantool instances.
+* ``stop``: stop one or more Tarantool instances.
+* ``status``: get the status of one or more current instances.
+* ``log``: get logs for one or more instances.
+* ``clean``: clean files for one or more instances.
+* ``pack``: pack the application into a distributable bundle.
+* ``repair``: patch cluster configuration files.
+* `admin <doc/admin.rst>`__: call an admin function provided by the application.
+* `replicasets <doc/replicasets.rst>`__: manage cluster replica sets running locally.
+* `enter <doc/connect.rst>`__ and `connect <doc/connect.rst>`__: connect to a running instance.
+* `failover <doc/failover.rst>`_: manage cluster failover.
```

Comment:
1. Why one underscore is changed to two underscores (I don't know RST well enough, sorry)? I see no difference on the document rendered by GitHub.
2. The last item still has one underscore: typo?